adafruit_is31fl3741/adafruit_ledglasses.py

```diff
@@ -30,86 +30,104 @@
 class Right_Ring(IS31FL3741):
     """The right eye ring of the LED glasses"""
 
+    ledmap = (
```

Comment:
plz also convert this to a b'' byte array, so it uses less RAM and should be faster lookups

Reply:
Some elements in table are > 255

Reply:
Later, yeah. I tried this and just messed it up.